plugin/src/main/resources/io/jenkins/plugins/analysis/core/steps/PublishIssuesStep/help-referenceJobName.html

```diff
@@ -7,7 +7,7 @@
     <p>
         If you are building pull requests or topic branches in separate jobs, it makes more sense that the plugin
         shows the new issues, that have been introduced by a pull request (or branch). In order to show these issues,
-        the reference needs to be set to the Jenkins job that builds the master branch (actually the job that contains
+        the reference needs to be set to the Jenkins job that builds the controller branch (actually the job that contains
```

Comment:
It's probably the "`main` branch" (instead of `controller`) following the suggestions from Git and GitHub.
(the same in other files below)

Reply:
There is also the term "primary branch" in [PrimaryBranchHealthMetric  (branch-api-plugin)](https://github.com/jenkinsci/branch-api-plugin/blob/f37d72dc42c5fd5968bdefe0b8b62f2b55342e5b/src/main/java/jenkins/branch/PrimaryBranchHealthMetric.java#L88) and [PrimaryInstanceMetadataAction (scm-api-plugin)](https://github.com/jenkinsci/scm-api-plugin/blob/041837a71e861a3cd7ade9269b2794b300c5ad47/src/main/java/jenkins/scm/api/metadata/PrimaryInstanceMetadataAction.java). That might be more portable to non-Git version control systems, if anyone is using Warnings NG with such.

Reply:
You are right, I missed that part. I think I can remove that help text anyway as it is not used anymore (deprecated property).

Reply:
I see, you removed it from the UI in <https://github.com/jenkinsci/warnings-ng-plugin/commit/d8cd90dbb42cbf5f466dd62908d062bc95a415db#diff-6e8f2bc6a01aead226824ce7fa43545955ac6f7d11cced6b1e739fbbe19b438a>.